NuGet.sln

```diff
@@ -624,7 +624,7 @@ Global
 		{50E33DA2-AF14-486D-81B8-BD8409744A38} = {08F523EC-3C2A-4A00-A54C-2E54C5AC856B}
 		{23725516-0A6E-48F9-A148-B5E3FA515B13} = {00CE0FBE-DA5A-4AD8-B0D6-04ECE6B04F0D}
 		{5039C2F9-CAD9-4162-849C-5EF7E3CE03D9} = {01BC4531-1E25-48D7-A8FD-A47D6FEC47FB}
-		{3DB4EDE3-B57B-498C-9EA9-A8BF8D9D4B2B} = {01BC4531-1E25-48D7-A8FD-A47D6FEC47FB}
+		{3DB4EDE3-B57B-498C-9EA9-A8BF8D9D4B2B} = {7323F93B-D141-4001-BB9E-7AF14031143E}
```

Comment:
Before this change NuGet.Credentials.Test showed up in Visual Studio under the NuGet.Client.UnitTests folder, even though the project is under test\NuGet.Core.Tests.  After this change, the project shows up under the NuGet.Core.UnitTests folder.